app/helpers/react_components/common/theme_toggle_button.rb

```diff
@@ -1,8 +1,17 @@
 module ReactComponents
   module Common
     class ThemeToggleButton < ReactComponent
+      initialize_with :disabled
```

Comment:
Normally it's best to have positive names for variables. So this might be better as `:enabled`. You might also want to do `enabled: true` (or `enabled: false`) as a default.